README.md

```diff
@@ -1,21 +1,121 @@
 # roave/you-are-using-it-wrong
 
-TBD
-
 [![Build Status](https://travis-ci.org/roave/you-are-using-it-wrong.svg?branch=master)](https://travis-ci.org/roave/you-are-using-it-wrong)
-[![Downloads](https://img.shields.io/packagist/dt/roave/you-are-using-it-wrong.svg)](https://packagist.org/packages/roave/you-are-using-it-wrong)
 [![Packagist](https://img.shields.io/packagist/v/roave/you-are-using-it-wrong.svg)](https://packagist.org/packages/roave/you-are-using-it-wrong)
-[![Dependencies](https://tidelift.com/badges/github/packagist/roave%2Fyou-are-using-it-wrong)](https://tidelift.com/subscription/pkg/packagist-roave%2Fyou-are-using-it-wrong?utm_source=packagist-ocramius%2Fpackage-versions&utm_medium=readme)
+
+This package enforces type checks during composer installation in downstream
+consumers of your package.
```

Comment:
Maybe worth emphasising that only usages of symbols declared in packages that directly depend on Roave/you-are-using-it-wrong will be checked in the root project, not the entire project. At first I thought that adding the dependency to my project would give me protection similar to what I would get by manually setting up Psalm.

Reply:
Will rephrase, thanks!